lstchain/visualization/plot_drs4.py

```diff
@@ -0,0 +1,88 @@
+
+from matplotlib import pyplot as plt
+
+import numpy as np
+from matplotlib.backends.backend_pdf import PdfPages
+from ctapipe.io import event_source
+from lstchain.calib.camera.r0 import LSTR0Corrections
+
+
+__all__ = ['plot_drs4',
```

Comment:
should be 'plot_waveforms'

Reply:
yes, actually I just modified this function adding the plot of other quantities (following the suggestions of Pawel and Julian)